runcomp.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+read -p "Defend Plus X (0 or 1): " defendPlusX
+read -p "Vision Channel (1,2 or Full): " visionChannel
+read -p "Specify Playbook file (default comp.pbk): " playbook
+
+make
+
+until ./run/soccer -pbk ${playbook:-"comp.pbk"} -defend ${defendPlusX:-"0"} -vision ${visionChannel:-"full"}; do
```

Comment:
Can you quote the variable dereferences here? For example:

`"${playbook:-"comp.pbk"}"`

It's good style to quote variable expansion in bash when possible (one reason is that it preserves whitespace in the argument). In this case it dosen't make too much of a difference though.